### DISCUSSION TITLE
Data Interface

### DISCUSSION THREAD

**Discussion by taaib** [m-ld/m-ld-spec#52] [Discussion]
What is the language used to interrogate the m-ld database?

**Comment by gsvarovsky** [comment]
It's called **json-rql**. It's a bit like SQL, in JSON form so that it's easy to author in code. You can explore starting here: https://js.m-ld.org/#transactions

There's also a bit more motivation and detail in this other Q&A: https://github.com/m-ld/m-ld-spec/discussions/48, including some thoughts on whether **m-ld** is really a _database_ at all 😉
